app/services/idv/send_phone_confirmation_otp.rb

```diff
@@ -80,22 +80,20 @@ def send_otp
       otp_sent_response
     end
 
-    def bucket
-      @bucket ||= AbTests::IDV_TEN_DIGIT_OTP.bucket(
-        idv_session.user_phone_confirmation_session.user.uuid,
-      )
-    end
-
     def format
-      return 'digit' if delivery_method == :voice && bucket == :ten_digit_otp
-
-      'character'
+      if delivery_method == :voice
+        'digit'
+      else
+        'character'
+      end
     end
 
     def length
-      return 'ten' if delivery_method == :voice && bucket == :ten_digit_otp
-
-      'six'
+      if delivery_method == :voice
+        'ten'
+      else
+        'six'
+      end
     end
```

Comment:
wondering if it would be easy to combine these methods? instead of having two methods with similar branches?

```ruby
format, length = code_format_length

def code_format_length
  case delivery_method
  when :voice
    ['ten', 'digit']
  else
    ['six', 'character']
  end
end
```

Reply:
Just fixed. Since these are only used in one method I went ahead and computed them in that method as well.